src/exportable/ExportableAddElementRule.ts

```diff
@@ -1,8 +1,15 @@
 import { fshrules } from 'fsh-sushi';
-import { Exportable } from './Exportable';
+import { repeat } from 'lodash';
+import { ExportableRule, INDENT_SIZE } from '.';
+
+export class ExportableAddElementRule extends fshrules.AddElementRule implements ExportableRule {
+  indent: number;
 
-export class ExportableAddElementRule extends fshrules.AddElementRule implements Exportable {
   constructor(path: string) {
     super(path);
   }
+
+  toFSH(): string {
+    return `${repeat(' ', INDENT_SIZE * (this.indent ?? 0))}${super.toFSH()}`;
```

Comment:
Currently, all of the `toFSH()` methods in SUSHI for rules return one FSH rule. The other consideration is cases where `toFSH()` will contain output with newlines. Currently, this is only the case for SUSHI's `ContainsRule` class, and goFSH's `ExportableContainsRule` doesn't call `super.toFSH()` since an `ExportableContainsRule` contains instances of `ExportableCardRule` and `ExportableFlagRule`.